content/synthetics/uptime_check.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Uptime Check
+kind: documentation
+beta: true
+description: Simulate and monitor HTTP requests from specific locations
+further_reading:
+- link: "https://www.datadoghq.com/blog/introducing-synthetic-monitoring/"
+  tag: "Blog"
+  text: "Introducing Datadog Synthetics"
+- link: "synthetics/"
+  tag: "Documentation"
+  text: "Manage your checks"
+- link: "synthetics/browser_check"
+  tag: "Documentation"
+  text: "Configure a Browser Check"
+---
+
+## Overview
+
+Uptime checks are HTTP requests (GET, POST, PUT, etc.) executed by Datadog to your web properties or application endpoints at configurable periodic intervals from multiple locations around the world. These checks verify not only that your applications are up and responding to requests, but that any conditions you define are met, such as response time, HTTP response code, and header or body contents.
+
+## Configuration
+### Request
+
+Define the request you want to be executed by Datadog:
+
+{{< img src="synthetics/uptime_check/uptime_check_make_request.png" alt="Uptime check make request" responsive="true" style="width:80%;">}}
+
+1. Choose the `Method` and `URL` to query. Available `Method` are:
+    * GET
+    * POST
+    * PATCH
+    * PUT
+    * DELETE
+    * OPTIONS
+
+2. Optional - Click on **Advanced options** to enrich your request with any custom Request Headers, Authentication credentials, body content, or cookies.
+
+3. Give a Name to your Uptime Check.
+4. Optional - Add tags to filter your Uptime Check in the [Synthetics page][1].
+
+5. Pick-up locations to run the test from, available locations are:
+    * Frankfurt (Request made from an AWS Datacenter)
+    * Tokyo (Request made from an AWS Datacenter)
+
+6. Choose a Check frequency between 1 run/min to 1 run/week.
```

Comment:
```suggestion
6. Choose a Check frequency between "1 run per minute" to "1 run per week".
```

This is different from https://github.com/DataDog/documentation/pull/3788/files#diff-0f7389d1d699727140bc0df31ce09ac1R36 - is that on purpose? Should they be the same?

Reply:
yup, uptime checks can be executed faster since it's "just" a ping and a check. Where a browser check scenario can potentially be really "long"